server/models/Profile.js

```diff
@@ -0,0 +1,53 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+    user: {
+        type: mongoose.Types.ObjectId,
+        ref: "User",
+        required: true
+    },
+    firstName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    lastName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    description: String,
+    gender: {
+        type: String,
+        required: true,
+        lowercase: true,
+        trim: true,
+        enum: ['male', 'female', 'other']
+    },
+    birthDate: Date,
+    phoneNumber: {
+        type: String,
+        trim: true
+    },
+    address: {
+        type: String,
+        required: true
+    },
+    availability: {
+        isAvailable: {
+            type: Boolean,
+            default: true
+        },
+        day: {
+            type: String,
+            required: true,
+            lowercase: true,
+            enum: ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
+        },
```

Comment:
Why this is required when it's possible to set availability to false?
`required` ie. an owner has availability attribute?
you can use [numbers]( https://momentjscom.readthedocs.io/en/latest/moment/02-get-set/06-day/)[0..6] or abbreviations instead.